### ISSUE TITLE
[APP SUBMITTED]: HTTP error None while loading URL http://thetvdb.com/api/F9C450E78D99172E/series/275143/en.xml

### ISSUE THREAD

**Issue by udochrist** [SickRage/SickRage#3606]
### INFO
Python Version: **2.7.13 (default, Dec 19 2016, 02:32:04) [GCC 4.7.3 20130102 (prerelease)]**
Operating System: **Linux-3.10.102-x86_64-with-glibc2.2.5**
Locale: None
Branch: **master**
Commit: SickRage/SickRage@4b3dad8eec3e238b1cadc809acf8e0e26deb9eb3
No Log available with ERRORS:
### ERROR
```
SHOWQUEUE-REFRESH :: [4b3dad8] theTVDB is down, can't use its data to add this show
```
---
_STAFF NOTIFIED_: @SickRage/owners @SickRage/moderators

**Comment by udochrist** [comment]
The error in itself is absolutely ok.

The use case here is thetvdb replying with a 403 http code (forbidden) to a request for a show that is not yet decided upon (only one ep produced up to now).

This should be logged as warning, not error.

**Comment by OmgImAlexis** [comment]
The title is meant to be like that so other reports end up here instead of making new issues each time.

**Comment by udochrist** [comment]
Thanks. I'm aware of that. 

Since I have that error for some weeks each night and my report didn't make it to other already filed issues I think it's safe to rename it however.

**Comment by sharkykh** [comment]
Did you add the show from Trending shows (Trakt) or Popular shows (IMDB)?

**Comment by udochrist** [comment]
No. Manual addition as a directory in my tv folder. 
The show is not added to sickrage at all. 

There is no .nfo or else in the directory. I assume the scanner picks it up when refreshing from tvdb on a nightly run and gets the 403. 
as long as the show is not confirmed thetvdb wants to prohibit any additions/changes and answers with 403

**Comment by sharkykh** [comment]
I agree that it shouldn't be an error. You said that the show isn't on SickRage - you can't open the shows page?

It sounds like it's stuck in limbo - you tried to add it, but it couldn't download the information so it got stuck in between?

Just so you know...
It responds with 404 (not found), not 403 (forbidden).
The reason being the show doesn't actually exist anymore on TheTVDB.com.

P.S.
If/when it gets added, my guess is that it will have a different ID, so you'll have to add it again.

**Comment by udochrist** [comment]
No, i only copied the show directory to the same folder that sr is using to store shows. I did not add it -  and I can't since it cannot be found on Tvdb.

Tvdb was returning 403s, now it started to return 404s.
I think they deleted the show again.


Von meinem iPad gesendet

**Comment by sharkykh** [comment]
What you're saying doesn't make much sense as SR only tries to update shows that are in its database (that you added).

Maybe you added it before they removed/locked it?
Have you tried moving the show folder out of the TV root dir to see if it makes any difference?

In any case, if you want, I can edit your sickbeard.db and remove this "ghost show" if it's in there.

**Comment by udochrist** [comment]
The show folder
```
six:TV Shows root$ ll /volume1/Media/TV\ Shows/Oasis
total 4
drwsrwsrwx 2 plex sc-media 4096 Apr  4 00:21 Season 01/
six:TV Shows root$ 
```

the database copied to my laptop...
```
boomer:~ udo$ sqlite3 sickbeard.db 
SQLite version 3.16.0 2016-11-04 19:09:39
Enter ".help" for usage hints.
sqlite> select * from tv_shows where location like '%Oasis%';
sqlite> 
```

**Comment by sharkykh** [comment]
Try looking for the indexer id instead (275143)

**Comment by udochrist** [comment]
that solved it. took me a while to understand i need to look at the url for the tvdb...

it is completely unrelated to the show i suspected. somehow thetvdb removed another show from its db at the time i added to the directory.

would it be possible to log the name of the show or indexer id or anything else to identify the show causing the error log?

**Comment by sharkykh** [comment]
I don't think it's possible to do that just yet, as we are using an older version of TheTVDB's API.
We should be able to do this once we start using the newer version.

To clarify, what I mean by "not possible" is,
the error we receive if a show is not found when searching TVDB using an show ID,
is **identical** to the error we receive when TheTVDB.com is unreachable. 

For now, even though it's a bit obscure, the error **does log the show ID** on TVDB.

`HTTP error None while loading URL http://thetvdb.com/api/F9C450E78D99172E/series/<SHOW-ID>/en.xml`
